doc/ug/system_list.md

```diff
@@ -9,4 +9,4 @@ Click on the design name to get more information about the design.
 |--------|---------------|--------------------|--------------|--------------|-------------|
 | [Earl Grey]({{< relref "hw/top_earlgrey/doc" >}}) | `top_earlgrey` | Verilator | ChipWhisperer CW310\* | *None yet.* | 0.1 release |
 
-`*` There exists a modified version of the Earl Grey top-level design that can be implemented on the Nexys Video FPGA board usable with a free EDA tool license.
+`*` There exists a modified version of the Earl Grey top-level design that can be implemented on the ChipWhisperer CW305 FPGA board usable with a free EDA tool license.
```

Comment:
I think `hw/top_earlgrey/ip/pinmux/doc/autogen/pinout_nexysvideo.md` needs to be `git rm`'ed. It does not get regenerated by `util/topgen.py -t hw/top_earlgrey/data/top_earlgrey.hjson -o hw/top_earlgrey/`, so that's fine, but it should be removed.

(Not related to this file/line, but I didn't see a better place for this comment.)

Reply:
Thanks for flagging this @andreaskurth . It seems that this file is out of date. It was last changed more than a year ago while the other pinout files have been regenerated some time in July last year. I've now added another commit to remove this file.